Playbooks/Send-UnhealthyAzureArcResourceAlert/azuredeploy.json

```diff
@@ -0,0 +1,215 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {

```

Comment:
Hello @mlaraibkhan please fill the required metadata

Reply:
You can refer to this playbook for metadata Solutions/AWSAthena/Playbooks/AWSAthenaPlaybooks/AWSAthena-GetQueryResults/azuredeploy.json

Reply:
Sure Thanks for pointing that out. I added the required metadata.